.eslintrc.json

```diff
@@ -1,3 +1,3 @@
 {
   "extends": ["next", "eslint:recommended"]
-}
```

Comment:
I think you removed the indent rule here. We need that as well.

Reply:
Added it back